developer-docs-site/docs/guides/nfts/mint-nft-cli.md

```diff
@@ -4,587 +4,291 @@ title: "Mint NFTs with the Aptos CLI"
+
+    aptos_token::create_collection(
+        creator,
+        string::utf8(COLLECTION_DESCRIPTION),
+        maximum_supply,
+        collection_name,
+        collection_uri,
+        MUTABLE_COLLECTION_DESCRIPTION,
+        MUTABLE_ROYALTY,
+        MUTABLE_URI,
+        MUTABLE_TOKEN_DESCRIPTION,
+        MUTABLE_TOKEN_NAME,
+        MUTABLE_TOKEN_PROPERTIES,
+        MUTABLE_TOKEN_URI,
+        TOKENS_BURNABLE_BY_CREATOR,
+        TOKENS_FREEZABLE_BY_CREATOR,
+        royalty_numerator,
+        royalty_denominator,
+    );
+    move_to(creator, MintConfiguration {
+        collection_name,
+        token_name,
+        token_uri,
+    });
+}
 ```
 
-4. As usual, receive output asking you to accept a gas fee for the transaction and enter: `yes`.
+First off, note the first couple lines of the function:
 
-5. Receive the output:
-
-```shell
-"Result": "Success"`
+```rust
+let creator_addr = signer::address_of(creator);
+assert!(creator_addr == @mint_nft, error::permission_denied(ENOT_AUTHORIZED));
 ```
 
-### Include admin functions
+This gates the ability to run this function to the owner of the contract. This isn't necessary, but unless you specifically design a contract around
+letting multiple accounts use it, it's simpler to only allow the account that deploys the contract to use it. This avoids a few potential vulnerabilities where you unintentionally allow an account to modify internal module resources for another account.
 
-See [create_nft_with_resource_and_admin_accounts.move](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/move-examples/mint_nft/3-Adding-Admin/sources/create_nft_with_resource_and_admin_accounts.move) to understand the related admind functions used:
+:::note
+We define `@mint_nft` when we deploy the module with the `--named-addresses mint_nft=<YOUR_ACCOUNT_ADDRESS>` flag.
+:::
 
+After that, we see the logic for our very basic initialize collection function. It creates a collection based off the parameters given in the
+function arguments and then stores the minting config info to the creator's account resources with `move_to`.
 
-* In struct `ModuleData`, we add two additional fields: `expiration_timestamp` and `minting_enabled`. These allow us to set and update when this collection will expire and enable or disable minting ad-hoc, respectively.
-* We included two admin functions - `set_minting_enabled()` and `set_timestamp()` - to update the `expiration_timestamp` and `minting_enabled` fields.
-* In the admin functions, we check if the caller is calling from the valid administrator's address. If not, we abort because the caller does not have permission to update the config of this module.
-* In `mint_event_ticket()`, we add two `assert` statements to make sure the user can mint tokens from this collection only if minting is enabled and the collection has not expired.
+Some of these fields are self-explanatory, but here's a quick rundown of what you're actually setting here:
```

Comment:
Might as well just link to the collection.move code for a detailed explanation of the fields?

Reply:
Yea, you're right. I think I was going to explain the fields more intuitively but ended up mostly just explaining what's already in collection.move

I'll leave the explanation of the royalty numerator/denominator and remove the rest